src/styles/about.scss

```diff
@@ -1,11 +1,11 @@
 @import 'learn.scss';
 
 .side-nav-about {
-  @extend .side-nav;
+  @extend %side-nav;
```

Comment:
I wonder if this one makes sense (orginally) given the wrestling you needed to do for the placeholder

Reply:
Not sure we got huge benefit in CSS bundle size after this change like [stylelint rule suggests](https://github.com/stylelint-scss/stylelint-scss/blob/master/src/rules/at-extend-no-missing-placeholder/README.md)

But anyway `learn.scss` looks more consistent now with parent selectors